queue.c

```diff
@@ -459,6 +462,9 @@ BaseType_t xQueueGenericReset( QueueHandle_t xQueue,
                     *ppucQueueStorage = ( uint8_t * ) pxQueue->pcHead;
                 }
 
+/* MISRA Ref 11.3.1 [Misaligned access] */
```

Comment:
Alignement issue.

Reply:
Thank you for your feedback. Fix in new commit.